include/avif/internal.h

```diff
@@ -158,33 +158,42 @@ typedef enum avifAlphaMultiplyMode
     AVIF_ALPHA_MULTIPLY_MODE_UNMULTIPLY
 } avifAlphaMultiplyMode;
 
-typedef struct avifReformatState
+typedef struct avifRGBSpaceInfo
```

Comment:
Nit: It is not clear what "space" means here. Especially in the `avifRGBSpaceInfo` struct, whose members are mostly sizes and offsets, one may think "space" means "buffer space". But I guess "space" means "color space", right?

This can be addressed by adding a comment or renaming the struct, e.g., `avifRGBInfo`, `avifRGBColorInfo`, `avifRGBColorSpace`, etc.

Reply:
Indeed I meant "color space". I will clarify.